bigquery/google/cloud/bigquery/table.py

```diff
@@ -1026,7 +1026,7 @@ def _configure_job_metadata(metadata,  # pylint: disable=too-many-arguments
         load_config['writeDisposition'] = write_disposition
 
 
-def _parse_schema_resource(info):
+def parse_schema_resource(info):
```

Comment:
Yes but the solution is not to add to the public interface, especially if we didn't think it was useful before the test-only change.